model_server.py

```diff
@@ -19,7 +19,7 @@
 )
 
 predictions_counter = prom_metrics.counter(
-    'predictions', 'The number of predictions.',
+    'predictions', 'The number of predictions.', labels={'count': 1}

```

Comment:
What does the `count`: 1 mean exactly?

Reply:
I wasn't sure if all Prometheus metrics need labels by default, so I made a dict with a static key-value pair. I don't think it's necessary though.